pkg/execution/state/redis_state/lua/queue/lease.lua

```diff
@@ -228,31 +234,41 @@ redis.call("ZADD", keyPartitionScavengerIndex, nextTime, item.id)
 -- so that expired leases can be re-processed.  We want to take the earliest time from the
 -- scavenger index such that we get a previously lost job if possible.
 -- TODO: Remove check on keyInProgressPartition once all new executors have rolled out and no more old items are in progress
-local concurrencyScores = redis.call("ZRANGE", keyInProgressPartition, "-inf", "+inf", "BYSCORE", "LIMIT", 0, 1, "WITHSCORES")
-local scavengerIndexScores = redis.call("ZRANGE", keyPartitionScavengerIndex, "-inf", "+inf", "BYSCORE", "LIMIT", 0, 1, "WITHSCORES")
+local concurrencyScores =
+	redis.call("ZRANGE", keyInProgressPartition, "-inf", "+inf", "BYSCORE", "LIMIT", 0, 1, "WITHSCORES")
+local scavengerIndexScores =
+	redis.call("ZRANGE", keyPartitionScavengerIndex, "-inf", "+inf", "BYSCORE", "LIMIT", 0, 1, "WITHSCORES")
 if scavengerIndexScores ~= false or concurrencyScores ~= false then
-  -- Either scavenger index or partition in progress set includes more items
-
-  local earliestLease = nil
-  if scavengerIndexScores ~= false and scavengerIndexScores ~= nil then
-    earliestLease = tonumber(scavengerIndexScores[2])
-  end
-
-  -- Fall back to in progress set
-  -- TODO: Remove this check once all items are tracked in scavenger index
-  if earliestLease == nil or (
-    concurrencyScores ~= false and
-    concurrencyScores ~= nil and
-    #concurrencyScores > 0 and
-    tonumber(concurrencyScores[2]
-  ) < earliestLease) then
-    earliestLease = tonumber(concurrencyScores[2])
-  end
-
-  if earliestLease ~= nil then
-    -- Ensure that we update the score with the earliest lease
-    redis.call("ZADD", concurrencyPointer, earliestLease, partitionID)
-  end
+	-- Either scavenger index or partition in progress set includes more items
+
+	local earliestLease = nil
+	if scavengerIndexScores ~= false and scavengerIndexScores ~= nil then
+		earliestLease = tonumber(scavengerIndexScores[2])
+	end
+
+	-- Fall back to in progress set
+	-- TODO: Remove this check once all items are tracked in scavenger index
+	if
+		earliestLease == nil
+		or (
+			concurrencyScores ~= false
+			and concurrencyScores ~= nil
+			and #concurrencyScores > 0
+			and tonumber(concurrencyScores[2]) < earliestLease
+		)
+	then
+		earliestLease = tonumber(concurrencyScores[2])
+	end
+
+	if earliestLease ~= nil then
+		-- Ensure that we update the score with the earliest lease
+		redis.call("ZADD", concurrencyPointer, earliestLease, partitionID)
+	end
+end
+
+-- report using throttle burst
+if usedThrottleBurst then
+	return 1
```

Comment:
looks unused 👍 

https://github.com/inngest/inngest/blob/1ecea27564ff4bc07b23b2f94ec0763af961abd4/pkg/execution/state/redis_state/queue.go#L2421-L2452

Reply:
no i meant it was unused on `main` so it's safe to use in your PR haha